scalar.go

```diff
@@ -46,6 +46,14 @@ func NewIdentifier(value string) *IdentifierInput {
 	}
 }
 
+func NewIdentifierList(values []string) []IdentifierInput {
```

Comment:
make sure to write a test for this in `scalar_test.go` so we can keep up code coverage.

Reply:
Ok, not sure if I'm testing it right but wrote something for this.  Open to any feedback though since I wasn't sure of best way to test it.